django_cryptography/core/signing.py

```diff
@@ -90,7 +91,7 @@ def loads(s, key=None, salt='django.core.signing', serializer=JSONSerializer, ma
 class Signer(object):
```

Comment:
This also seems to deviate from parity with [django.core.signing.Signer](https://github.com/django/django/blob/master/django/core/signing.py#L152)